snap/snapcraft.yaml

```diff
@@ -215,7 +218,8 @@ parts:
   containerd:
     plugin: make
     source: https://github.com/containerd/containerd.git
-    # from https://github.com/moby/moby/blob/docker-v29.1.2/Dockerfile#L166
+    # fetch from `docker version` installing the docker deb packages
+    # from docker archive (https://docs.docker.com/engine/install/ubuntu/)
```

Comment:
This and other comments raise a few questions. How was the version "fetched"? Is it fetched dynamically during the build? Assuming one figure out that the comment meant to say the "version is fetched by running `docker version` installed via the deb package from docker archive", the next questions would be: which Docker version? when was this done? 

Let's rather remove all the comments and create a subsection in the README under Development to explain how one can upgrade the snapcraft.yaml using the helper script, what gets updated and what doesn't.